src/language-server/config/cache-busting-resolver.js

```diff
@@ -0,0 +1,65 @@
+// @ts-check
+const { pathToFileURL } = require("node:url");
+
+/** @import { ResolveContext, ResolutionResult, LoadResult, ImportContext } from "./cache-busting-resolver.types" */
+
+/**
+ * @param {string} specifier
+ * @returns {string}
+ */
+function bustFileName(specifier) {
+  const url = pathToFileURL(specifier);
+  url.pathname = url.pathname + "." + Date.now() + ".js";
+  return url.toString();
+}
+
+/**
+ *
+ * @param {string} specifier
+ * @param {ResolveContext} context
+ * @param {(specifier: string,context: ResolveContext) => Promise<ResolutionResult>} nextResolve
+ * @returns {Promise<ResolutionResult>}
+ */
+async function resolve(specifier, context, nextResolve) {
+  if (context.importAttributes.as !== "cachebust") {
```

Comment:
It was ⁠`context.importAssertions`. So, in some older versions of Node.js, `context importAttributes` will be `undefined`. Unfortunately, Cursor is using Node.js 20.9.0 in their latest version.

See, https://nodejs.org/api/module.html#hooks:~:text=v21.0.0%2C%20v20.10.0%2C%20v18.19.0